components/devtools_traits/lib.rs

```diff
@@ -252,8 +252,7 @@ bitflags! {
 
 #[derive(Deserialize, Serialize)]
 pub struct PageError {
-    #[serde(rename = "type")]
-    pub type_: String,
+    pub _type: String,
```

Comment:
Do `#[serde(rename = "type_")]`, undescore as a prefix has special meaning in Rust (silences dead_code warnings).